### PR TITLE
Use T! macro for symbolic tokens

### PR DESCRIPTION
**Pull request by matklad**
That's just a syntactic change, but I think it makes the code a tad more readable.

### CODE REVIEW

File: src/rules.rs [hunk 1]
```diff
@@ -624,7 +624,7 @@ pub(crate) fn indentation() -> IndentDsl {
             "#)
         .rule("Indent binops top level")
             .inside(p(NODE_BIN_OP) & p(on_top_level))
-            .not_matching(p(TOKEN_CONCAT) | p(VALUES))
+            .not_matching(p(T![++]) | p(VALUES))
             .set(Indent)
             .test(r#"
                 {
```
Comment (Rizary) [inline src/rules.rs:630]: I think it got deleted in https://github.com/nix-community/nixpkgs-fmt/pull/209/files#diff-a22d0d260e63d7c19b16d3ff4313fd28L633-L647

Comment (matklad) [reply src/rules.rs:630]: Yeah, but I'd rather merge this PR first, and then rebase that one, as it is more of an experiment at this point

Comment (Rizary) [reply src/rules.rs:630]: Ah yeah you're right, that is make sense.

**Pull request merged**